app/assets/javascripts/ui_components/toolbar.coffee

```diff
@@ -0,0 +1,10 @@
+$ ->
+  # all checked checkboxes display as active
+  $(".toolbar .checkbox input[type='checkbox']:checked").parents('.checkbox').addClass('active')
```

Comment:
Hmm, I see. Then I guess we have no other choice but to keep a bit of jquery. 

We can simplify it a bit, though:

```coffeescript
$ ->
  $('.toolbar .checkbox input').change ->
    $(this).parents('.checkbox').toggleClass('active', this.checked)

  $('.toolbar .checkbox input').trigger('change')
```